ci/jenkins_build.sh

```diff
@@ -0,0 +1,126 @@
+export HELM_INSTALL_TIMEOUT="${HELM_INSTALL_TIMEOUT:-180}"
+export HELM_TEST_LOGGING="${HELM_TEST_LOGGING:-true}"
+export HELM_VERSION="${HELM_VERSION:-3.1.3}"
+export KUBECTL_VERSION="${KUBECTL_VERSION:-1.16.9}"
+export SKIP_GCLOUD_LOGIN="${SKIP_GCLOUD_LOGIN:-false}"
+# Use release name not based on timestamp since tests can run in parallel
+export RELEASE_NAME="$CONJUR_NAMESPACE"
+
+announce "Building gcloud/kubectl/helm client image..."
+tmp_dir="$(pwd)/.tmp"
+bin_dir="${tmp_dir}/bin"
+tools_image_name="conjur-oss-helm-kubectl"
+
+mkdir -p "${bin_dir}"
+mkdir -p "${tmp_dir}/.kube"
+mkdir -p "${tmp_dir}/.config"
+
+# Create a local alias for running 'gcloud' in client container
+cat > "${bin_dir}/gcloud" <<EOF
+  docker run --rm \
+    --ipc="host" \
+    -v "${tmp_dir}/.kube:/root/.kube" \
+    -v "${tmp_dir}/.config:/root/.config" \
+    --entrypoint /usr/bin/gcloud \
+    "${tools_image_name}" \
+    "\$@"
+EOF
+chmod +x "${bin_dir}/gcloud"
+
+# Create a local alias for running 'kubectl' in client container
+cat > "${bin_dir}/kubectl" <<EOF
+  docker run --rm \
+    -v "${tmp_dir}/.kube:/root/.kube" \
+    --entrypoint /usr/local/bin/kubectl \
+    "${tools_image_name}" \
+    "\$@"
+EOF
+chmod +x "${bin_dir}/kubectl"
+
+# Create a local alias for running 'helm' in client container
+cat > "${bin_dir}/helm" <<EOF
+  docker run --rm \
+    -v "${tmp_dir}/.kube:/root/.kube" \
+    -v "$(cd ../conjur-oss; pwd):/root/conjur-oss" \
+    --entrypoint /usr/local/bin/helm \
+    -w /root \
+    "${tools_image_name}" \
+    "\$@"
+EOF
+chmod +x "${bin_dir}/helm"
+
+export PATH="${bin_dir}:${PATH}"
+
+if ! is_helm_v2; then
+  # Helm v3 requires units for timeout values
+  HELM_INSTALL_TIMEOUT+="s"
+fi
+
+# Build the gcloud/kubectl/helm client container image
+#docker pull google/cloud-sdk
+docker build -t "${tools_image_name}" \
+             --quiet \
+             --build-arg HELM_VERSION="$HELM_VERSION" \
+             --build-arg KUBECTL_VERSION="$KUBECTL_VERSION" \
+             -f Dockerfile \
+             .
+
+if [ "$SKIP_GCLOUD_LOGIN" = true ]; then
+  # Copy kubectl and gcloud config
+  cp $HOME/.kube/config "${tmp_dir}/.kube"
+  cp -r $HOME/.config/gcloud "$tmp_dir"
+else
+  announce "Logging in to GCP..."
+  ./platform_login.sh
+
+  # Fix permissions on files created within Docker
+  docker run --rm \
+             -v "${tmp_dir}/.kube:/root/.kube" \
+             -v "${tmp_dir}/.config:/root/.config" \
+             "${tools_image_name}" \
+             bash -c "chown ${UID} -R /root/.kube/config /root/.config/*"
+fi
+
+function delete_namespace {
+  announce "Deleting namespace $CONJUR_NAMESPACE"
+  kubectl delete namespace --ignore-not-found=true "$CONJUR_NAMESPACE"
+}
+
+announce "Deploying and testing Conjur OSS"
+cd ..
+trap delete_namespace EXIT
+./test.sh
+if [ $? -eq 0 ]; then
+  announce "                 SUCCESS"
+else
+  announce "                 FAILED"
+fi
```

Comment:
This if/else block will never run - you have the `set -e` flag which will make `./test.sh` exit the script on failure. Even if it did run, the error would not propagate to jenkins.